compose/cli/main.py

```diff
@@ -496,7 +500,8 @@ def rm(self, options):
                     or yesno("Are you sure? [yN] ", default=False):
                 self.project.remove_stopped(
                     service_names=options['SERVICE'],
-                    v=options.get('-v', False)
+                    v=options.get('-v', False),
+                    one_off=options.get('--all')
```

Comment:
There's a bit of a mismatch here, where you do the `None if all else False` in the cli layer for one call, but it's done in the project layer for the other. I think this should be consistent.

I'm usually pretty weary of the True/False/None pattern. I think it's pretty non-intuitive.  Could we use another enum here (we have a bunch already in `compose/service.py` using enum34)?

The values could be: `include` (currently None), `exclude` (currently False), `only` ( currently True).